### ISSUE TITLE
Building to run on Windows 10

### ISSUE THREAD

**Issue by bmwenger007** [e-ago/bitcracker#14]
I am having a very difficult time getting this to build on Windows.  I have set the path to the make.exe file and the cl.exe file and it still will not work.  I have also tried putting them into the src_XX folder and then running make and it fails when running. I have installed Cygwin to get the make and gcc executables.  I have installed Visual Studio 2017 and the CUDA toolkit version 10.  Anything else I may be missing (other than a decent Linux computer on hand?)?

Thanks,

**Comment by bmwenger007** [comment]
As of today, I have been able to get the hashs (both User Password and Recovery Key) but I can not get Bitcracker to build the main portion.  I have tried using JtR, but I keep having the same issues and I can't find what I'm missing. Are there step by step instructions somewhere I can use to compile this? I have a 4 GB flash drive that my friend wants the data off of, but he can't remember the password.
